app/src/main/java/org/luxoft/sdl_core/BleWriter.java

```diff
@@ -0,0 +1,15 @@
+package org.luxoft.sdl_core;
+
+public abstract class BleWriter {
```

Comment:
@ychernysheva please replace with interface

Reply:
Done in 8effe4942498195cce7a335540df658a92a3e296